bin/picca_coadd_zint.py

```diff
@@ -0,0 +1,458 @@
+#!/usr/bin/env python3
+"""Coadd correlation function from different redshift intervals"""
+import os
+import argparse
+import fitsio
+import numpy as np
+
+from picca.utils import userprint
+
+
+def coadd_correlations(fi,fout):
+    """Coadds correlation functions measured in different redshift intervals.
+
+    Args:
+        fi: list of strings
+            List of the paths to the input correlations to be coadded.
+        fout: string
```

Comment:
Similarly this could be renamed to `output_file`

Reply:
Agreed, I'll address this in a pull shortly